packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,8 +202,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
-                    allowedValues: options.map(option => option.value)
+                    transform: value => options.find(option => option.value.toString() === value).label,
+                    allowedValues: options.map(option => option.value.toString())
```

Comment:
It shouldn't happen here, at schema level, because `option.value` might be an object. I know, that AntD disallows to use non-strings as options, but it should happen at the component level then.